src/js/components/Markdown/doc.js

```diff
@@ -11,9 +11,12 @@ export const doc = Markdown => {
       <Markdown>{content}</Markdown>`,
     );
 
-  // DocumentedMarkdown.propTypes = {
-  //   content: PropTypes.string.description('The markdown text to render.'),
```

Comment:
I don't see the content anywhere, have we decided it is no longer needed?

Reply:
Well it was commented out so i thought it should be deleted. If you prefer i can let the comment there.